src/types.ts

```diff
@@ -214,7 +265,15 @@ export interface PostHogConfig {
     feature_flag_request_timeout_ms: number
     get_device_id: (uuid: string) => string
     name: string
+    /**
+     * this is a read-only function that can be used to react to event capture
+     */
     _onCapture: (eventName: string, eventData: CaptureResult) => void
+    /**
+     * This function - if provided - is called immediately before sending data to the server.
+     * It allows you to edit data before it is sent, or choose not to send it all.
```

Comment:
Not really immediately before sending but rather before enqueuing unless you flip the send_mmediately flag right?

Reply:
Feels like `sanitize_properties` would also be redundant.

Reply:
> Not really immediately before sending but rather before enqueuing unless you flip the send_mmediately flag right?

yep, went with hopefully clear to the user over technically correct 🙈